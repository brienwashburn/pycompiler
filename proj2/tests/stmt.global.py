global a
global a, b, c
global q, r, sdfa
