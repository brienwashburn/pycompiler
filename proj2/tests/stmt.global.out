(Module (Global a) (Global a b c) (Global q r sdfa))
